src/mavoscript.js

```diff
@@ -233,22 +230,19 @@ var _ = Mavo.Script = {
 			precedence: 6
 		},
 		"neq": {
-			logical: true,
+			comparison: true,
 			scalar: (a, b) => a != b,
 			symbol: ["!="],
 			identity: true
 		},
 		"and": {
-			logical: true,
-			scalar: (a, b) => !!a && !!b,
-			identity: true,
+			scalar: (a, b) => a && b,
+			identity: false,
```

Comment:
Why are we changing the identity of `and` to `false`? That doesn't seem to make sense or be related to this PR.
Everything else looks ok.

Reply:
Oh ya my bad, that's not related to this PR.